test/unit/protocol/message_layer/StreamMessageV30.test.js

```diff
@@ -128,7 +128,7 @@ describe('StreamMessageV30', () => {
                 foo: 'bar',
             }
             const msg = new StreamMessageV30(
-                ['streamId', 0, Date.now(), 0, 'publisherId', 1], [1529549961000, 0], StreamMessage.CONTENT_TYPES.MESSAGE, content,
+                ['streamId', 0, Date.now(), 0, 'publisherId', '1'], [1529549961000, 0], StreamMessage.CONTENT_TYPES.MESSAGE, content,
```

Comment:
not int? 👍  :) https://github.com/streamr-dev/streamr-client-protocol-js/blob/master/PROTOCOL.md#messageid